### PR TITLE
Add Japanese Translation

### PR DESCRIPTION
**Pull request by yoshioterada**
This is a translation for Japanese.
- [x] README (main)
- [x] README (lesson 00)
- [x] README (lesson 01)
- [x] README (lesson 02)
- [x] README (lesson 03)
- [x] README (lesson 04)
- [x] README (lesson 05)
- [x] README (lesson 06)
- [x] README (lesson 07)
- [x] README (lesson 08)
- [x] README (lesson 09)
- [x] README (lesson 10)
- [x] README (lesson 11)
- [x] README (lesson 12)
- [x] README (lesson 13)

### CODE REVIEW

File: 00-course-setup/translations/ja-jp/README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+重要なお知らせ：このリポジトリ内に含まれるテキストを翻訳する場合、機械翻訳の使用は避けてください。翻訳の内容はコミュニティで検証するため、ご自身が熟知している言語でのみ翻訳作業にご協力ください。
+
+プル・リクエストを提出すると、CLA-bot が自動的に CLA（コントリビューター・ライセンス契約）の提出が必要かどうかを判断し、プル・リクエストに適切なマーク（例：ラベル、コメント）を付けます。bot の指示にお従いください。この処理は、CLA を使用するすべてのリポジトリで一度だけ行う必要があります。
+
+このプロジェクトは、[Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/?WT.mc_id=academic-105485-yoterada) を採用しています。詳細は、Code of Conduct FAQ をご覧いただくか、追加の質問やコメントがある場合は、[メールで opencode](opencode@microsoft.com) にお問い合わせください。  
+
+## それでは、始めましょう
+
+Now that you have completed the needed steps to complete this course, let's get started by getting an [introduction to Generative AI and LLMs].
+
```
Comment (koudaiii) [inline 00-course-setup/translations/ja-jp/README.md:139]:
```suggestion
```

You've already translated this part into Japanese at line 140, so I think the original text here isn't necessary. 😄

File: 12-designing-ux-for-ai-applications/README.md [hunk 2]
```diff
@@ -1,120 +1,121 @@
-# Designing UX for AI Applications
```
Comment (koudaiii) [inline 12-designing-ux-for-ai-applications/README.md:1]: @yoshioterada It looks like you've translated section 12. directly from the original text. Is this correct?

File: 01-introduction-to-genai/translations/ja-jp/README.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+1. 毎回まったく同じ回答が返ってきます
+1. 物事を完璧に行い、計算をしたり、正しく動作するコードを生成するのに最適です
+1. 同じプロンプトを使用しても、回答が異なる場合があります。また、テキストやコードなど最初に何らかのヒントを得るために利用するのには最適です。しかし、得られた結果は改善する必要があります。
+
+A: 3, 大規模言語モデル（LLM）は決定的ではなく、出力結果にはばらつきがありますが、温度設定を調整することで、ばらつきを制御することが可能です。完璧な結果を期待するべきではありません。LLM は皆様の負担を軽減するために存在し、物事を始める際に最初の良い出発点を提供します。 ただ多くの場合、段階的に改善していく必要があるでしょう。
+
+## お疲れ様でした! 次のレッスンを続ける
+
+このレッスンを修了後、[Generative AI Learning collection](https://aka.ms/genai-collection?WT.mc_id=academic-105485-yoterada) に進んで、生成系 AI の知識をさらに深めましょう!
```
Comment (koudaiii) [inline 01-introduction-to-genai/translations/ja-jp/README.md:148]: I found various translation patterns for this link: https://aka.ms/genai-collection?WT.mc_id=academic-105485-yoterada. It seems like the following. Is there any reason behind this? 🤔  '生成系 AI 学習コレクション' appears to be the most used in this pull request. 😄 

- Generative AI Learning collection
- 生成系 AI 学習コレクション
- 生成型AI学習コレクション

File: 08-building-search-applications/translations/ja-jp/README.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+セマンティック検索とは、検索クエリ内の単語の意味、つまりセマンティクスを利用して、関連性のある結果を返す検索手法のことを指します。
+
+セマンティック検索の一例を挙げてみましょう。あなたが車を購入したいと考えていて、「私が将来乗りたい夢の車」を検索したとします。この場合、セマンティック検索は、あなたが車について「夢」見ているわけではなく、「理想の車」を購入したいという意図を理解します。そして、その意図に関連する結果を返します。これに対して、「キーワード検索」は文字通りに車についての夢を検索し、結果として関連性のない情報を返すことが多くなります。
+
+## テキスト埋め込み (Embeddings) とは何でしょうか？
+
+[テキストの埋め込み (Embeddings)](https://en.wikipedia.org/wiki/Word_embedding?WT.mc_id=academic-105485-yoterada) は、[自然言語処理](https://en.wikipedia.org/wiki/Natural_language_processing?WT.mc_id=academic-105485-yoterada)において用いられるテキストの表現方法で、テキストの意味を数値的に表現したものです。この埋め込みは、機械が理解しやすいように数値データとして表現するために使用します。テキスト埋め込みを生成するための AI モデルは多数存在しますが、今回のレッスンでは OpenAI が提供する埋め込みモデルを使用して埋め込みを生成する方法に焦点を当てます。
+
+Here's an example, imagine the following text is in a transcript from one of the episodes on the AI Show YouTube channel:
```
Comment (koudaiii) [inline 08-building-search-applications/translations/ja-jp/README.md:50]:
```suggestion
```

You've already translated this part into Japanese at [line 52](https://github.com/microsoft/generative-ai-for-beginners/pull/230/files#diff-420250f0802e53e42481895d4d7a925e8c9806704711948e6cc717b0290e07b7R52) , so I think the original text here isn't necessary. 😄

File: 04-prompt-engineering-fundamentals/translations/ja-jp/README.md [hunk 5]
```diff
@@ -0,0 +1,396 @@
...
+
+- **管理者**: AI に _カリキュラムのデータを分析し、カバーしてない領域を特定する_ よう要求する可能性があります。AI はその結果をまとめたり、コードを用いて可視化することができます。
+- **教員**: AI に _特定の生徒と授業科目に応じた授業計画を作成する_ 可能性があります。AI は指定されたフォーマットに従って個別に計画を作成できます。
+- **生徒**: AI に _苦手な科目で個別指導を依頼する_ 可能性があります。AI は生徒のレベルに合わせたレッスン、ヒント、例を提供して指導することができます。
+
+これらは、ほんの一例です。教育専門家から厳選された、オープンソースのプロンプト・ライブラリ「[Prompts For Education](https://github.com/microsoft/prompts-for-edu/tree/main?WT.mc_id=academic-105485-yoterada)」を確認し、さらに広い視野で可能性を探ってみてください！ _サンドボックスでそれらのプロンプトを試しに実行したり、OpenAI Playground で試してどのような結果が出力されるか試してください！_  
+
+
+<!--
+LESSON TEMPLATE:
```
Comment (koudaiii) [inline 04-prompt-engineering-fundamentals/translations/ja-jp/README.md:51]: This 'LESSON TEMPLATE:' seems like it's meant as a note for instructors. I've counted at least five instances. Do you plan not to translate this section? 🤔 

https://github.com/microsoft/generative-ai-for-beginners/pull/230/files#diff-03ee89d324536f00c2c930a03b7bd758590d83d23590c31192f561099e4166b3R50-R58

https://github.com/microsoft/generative-ai-for-beginners/pull/230/files#diff-03ee89d324536f00c2c930a03b7bd758590d83d23590c31192f561099e4166b3R169-R177

https://github.com/microsoft/generative-ai-for-beginners/pull/230/files#diff-03ee89d324536f00c2c930a03b7bd758590d83d23590c31192f561099e4166b3R295-R304

https://github.com/microsoft/generative-ai-for-beginners/pull/230/files#diff-03ee89d324536f00c2c930a03b7bd758590d83d23590c31192f561099e4166b3R337-R346

https://github.com/microsoft/generative-ai-for-beginners/pull/230/files#diff-03ee89d324536f00c2c930a03b7bd758590d83d23590c31192f561099e4166b3R372-R375

File: 11-integrating-with-function-calling/translations/ja-jp/README.md [hunk 6]
```diff
@@ -0,0 +1,450 @@
...
+        url = module["url"]
+        results.append({"title": title, "url": url})
+     return str(results)
+   ```
+
+   `functions` 変数に記載した関数名にマップする、実際の Python 関数を作成する方法に注目してください。また、必要なデータを取得するために、実際に外部 API 呼び出しも行っています。ここでは、Microsoft Learn API に対してトレーニング モジュールを検索しています。
+
+いま、`functions` 変数とそれに対応する Python 関数を作成しましたが、実際の Python 関数を呼び出すために、これら 2 つをマッピングした内容を、LLM に指示するにはどうすればよいでしょうか？
+
+1. To see if we need to call a Python function, we need to look into the LLM response and see if `function_call` is part of it and call the pointed out function. Here's how you can make the mentioned check below:
```
Comment (koudaiii) [inline 11-integrating-with-function-calling/translations/ja-jp/README.md:330]:
```suggestion
```

You've already translated this part into Japanese at [line 331](https://github.com/microsoft/generative-ai-for-beginners/pull/230/files#diff-653d8b1f89858cccd0d260e78b0f6051f3dec8ffc975554f7ef01e5d20791629R331) , so I think the original text here isn't necessary. 😄

**Comment by satonaoki** [comment]
@yoshioterada 

取り急ぎ。全編を通して、「生成系 AI 」を「生成 AI 」に。

時間が取れれば、後でレビューします。

**Review by carlotta94c — APPROVED** [review]

**Pull request merged**